src/core/index.js

```diff
@@ -183,11 +200,11 @@ let Core = function( opts ){
       }
 
       // bind all the ready handlers registered before creating this instance
-      for( let i = 0; i < readies.length; i++ ){
-        let fn = readies[ i ];
+      for( let i = 0; i < options.reg.readies.length; i++ ){
```

Comment:
@maxkfranz 
When I call ```mount()```, do I need to:
* call the ready functions associated with the container?
* call ```cy.startAnimationLoop()```

Reply:
Ready doesn't have anything *directly* to do with rendering: It's about when the data is loaded and the layout has finished, because either could be async.   *Indirectly*, ready happens around the same time as -- but a bit before -- the first frame.

You will have to handle the animation loop, in case you went from a headless instance (with `styleEnabled: false`) to a rendered one.  You'll need to stop the loop at the start of remounting and then start a new one towards the end of remounting:  Rendered instances need to hook into the renderer's render loop to sync animation calculations just before rendering each frame.